src/main/java/org/jenkinsci/plugins/changeassemblyversion/ChangeAssemblyVersion.java

```diff
@@ -170,13 +241,14 @@ public boolean perform(AbstractBuild build, Launcher launcher, BuildListener lis
             StringWriter sw = new StringWriter();
             ex.printStackTrace(new PrintWriter(sw));
             listener.getLogger().println(sw.toString());
-            return false;
+
+            throw new AbortException(sw.toString());
         }
-        return true;
+        
     }
-
+    
     @Extension
-    public static class Descriptor extends BuildStepDescriptor<Builder> {
+    public static class DescriptorImpl extends BuildStepDescriptor<Builder> {
```

Comment:
Why rename?

Reply:
Without this rename, the plugin doesn't show in list in "Pipeline Syntax"

Reply:
Without this rename, the plugin doesn't show in list in "Pipeline Syntax"